lib/IRGen/IRGenDebugInfo.cpp

```diff
@@ -298,7 +298,9 @@ class IRGenDebugInfoImpl : public IRGenDebugInfo {
       if (auto generatedInfo = SM.getGeneratedSourceInfo(BufferID)) {
         // We only care about macros, so skip everything else.
         if (generatedInfo->kind != GeneratedSourceInfo::ReplacedFunctionBody &&
-            generatedInfo->kind != GeneratedSourceInfo::PrettyPrinted)
+            generatedInfo->kind != GeneratedSourceInfo::PrettyPrinted &&
+            generatedInfo->kind != GeneratedSourceInfo::DefaultArgument &&
+            generatedInfo->kind != GeneratedSourceInfo::Attribute)
```

Comment:
The comment above says "We only care about macros, so skip everything else", but aren't macros attributes in the grammar? I wouldn't expect us to skip `GeneratedSourceInfo::Attribute` here.

Reply:
The comment means "we only care about macro expansion buffers." I'll clarify it